src/FNA3D_Driver_D3D11.c

```diff
@@ -2699,6 +2706,10 @@ static void D3D11_INTERNAL_CreateSwapChain(
 			{
 				colorSpace = DXGI_COLOR_SPACE_RGB_FULL_G10_NONE_P709;
 			}
+			else if (sRGB)
+			{
+				colorSpace = DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709;
```

Comment:
Doesn't the `else` cover this?

Reply:
I'm not sure this or the other else are both the correct color space

Reply:
I did some more testing and it seems like they're right, but it feels like either the sRGB or non-sRGB arm should be G10 so I'm not sure what's going on.